macro/src/lib.rs

```diff
@@ -324,13 +325,6 @@ fn gen_bridge(mut input: ItemMod) -> ItemMod {
             // never referenced. #[diplomat::transparent_convert] handles adding repr(transparent)
             // on its own
             if !info.opaque {
-                let copy = if !info.is_out {
-                    // Nothing stops FFI from copying, so we better make sure the struct is Copy.
-                    quote!(#[derive(Clone, Copy)])
-                } else {
-                    quote!()
-                };
```

Comment:
Question: Why this change?

Reply:
DiplomatOption cannot implement Copy, unfortunately, because it has a custom Drop impl. There's currently no way in Rust to have "Drop only for nontrivial fields" aside from the default drop glue.

The only thing requiring Copy was this sanity check. There are now Copy types allowed in input structs, so the sanity check is now incorrect and needed to be removed.

Reply:
> There are now Copy types allowed in input structs, so the sanity check is now incorrect and needed to be removed.

What?

Reply:
sorry, non-Copy types. DiplomatOption isn't Copy